IPs/RSKIP146.md

```diff
@@ -0,0 +1,134 @@
+# Bridge events data and formatting
+
+|RSKIP          |146           |
+| :------------ |:-------------|
+|**Title**      |Bridge events data and formatting |
+|**Created**    |25-OCT-19 |
+|**Author**     |MI & JD |
+|**Purpose**    |USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+## Abstract
+
+This RSKIP proposes redefining Bridge events encoding format and the data they include.
+
+## Motivation
+
+The Bridge precompiled contract has several events. Those events are formatted using RLP, which renders them incompatible with the existing web3/solidity tooling (explorer included).
+Converting them to a compatible format would help users to easily get the data from those events.
+
+In addition to this, the current specification of the data included within these events is scarce and, at the same time, doesn't expose as much data as potential use cases may need.
+
+We propose to redefine the events and their data, to provide the means for a user to track the flow of the critical processes, these being lock, release and federation change.
```

Comment:
Add a full stop here **We propose to redefine the events and their data. To provide the means...**

Also, I wouldn't use _**We**_. Maybe _**The proposal is to redefine...**_